src/app/components/common/topbar/topbar.component.ts

```diff
@@ -263,8 +263,9 @@ export class TopbarComponent extends ViewControllerComponent implements OnInit,
       this.target.next({ name: 'SubmitComplete', sender: this });
       this.isWaiting = false;
     }
+    const setupComplete = localStorage.getItem('setupComplete') as unknown as boolean;
```

Comment:
The reason you got a type error is because localstorage actually contains a string.
The check below will evaluate to true for any string, even `'false'`.

Reply:
Could not reproduce this but I have seen it happen. The code is pretty straight forward so adding another check was the only solution I could think of.